src/web/common.rkt

```diff
@@ -56,13 +56,13 @@
     ("C" . ,(curryr core->c "code"))))
 
 (define (render-preprocess-struct preprocess)
-  (define vars (string-append "[" (string-join (map symbol->string (symmetry-group-variables preprocess)) " ") "]"))
-  (render-large vars "=" (string-append "sort(" vars ")")))
+  (define vars (string-append "[" (string-join (map symbol->string (symmetry-group-variables preprocess)) " \\; ") "]"))
```

Comment:
Can we put commas between elements instead of separating them with a hard space?